Client/OrganizationsApi.cs

```diff
@@ -141,7 +141,8 @@ public async Task<Organization> FindOrganizationByIdAsync(string orgId)
         /// <returns>List all organizations</returns>
         public async Task<List<Organization>> FindOrganizationsAsync()
         {
-            return await _service.GetOrgsAsync().ContinueWith(t => t.Result.Orgs);
+            var result = await _service.GetOrgsAsync();
+            return result.Orgs;
```

Comment:
Good point! We will fix it in whole client - see https://github.com/influxdata/influxdb-client-csharp/issues/108.

Could you please remove it from this PR.